### PR TITLE
Readme: Add upgrade instructions

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,19 @@ the steps accordingly.
...
+```
+cd /opt/ff-node-monitor/src/
+sudo systemctl stop ff-node-monitor
+git pull
+sudo -u ff-node-monitor /opt/ff-node-monitor/.cargo/bin/rustup default $(cat rust-version)
+sudo -u ff-node-monitor /opt/ff-node-monitor/.cargo/bin/cargo build --release
+sudo systemctl start ff-node-monitor
+```
+
 ### Integration
```
Comment (RalfJung) [inline README.md]: This should probably also become a 2nd-level header then?

File: README.md [hunk 2]
```diff
@@ -129,6 +129,19 @@ the steps accordingly.
 
 That's it!  The service should now be running and working.
 
+## Upgrade
+
+Upgrade the service to the latest git version with these stepos:
```
Comment (RalfJung) [inline README.md]: Typo: "stepos"

File: README.md [hunk 3]
```diff
@@ -129,6 +129,19 @@ the steps accordingly.
...
+
+```
+cd /opt/ff-node-monitor/src/
+sudo systemctl stop ff-node-monitor
```
Comment (RalfJung) [inline README.md]: Actually this is not necessary. If you replace this by `sudo rm /opt/ff-node-monitor/src/target/release/ff-node-monitor`, you can change the `start` below to a `restart` and reduce the downtime.

**Comment by rubo77** [comment]
done

File: README.md (hunk 1 above)
Comment (rubo77) [reply README.md]: done

**Comment by RalfJung** [comment]
Thanks :)

**Pull request merged**